test/lit.cfg

```diff
@@ -1420,4 +1416,30 @@ if platform.system() == 'Linux':
         config.available_features.add("LinuxDistribution=" + distributor + '-' + release)
         lit_config.note('Running tests on %s-%s' % (distributor, release))
 
+if run_vendor == 'apple':
+    config.available_features.add('libdispatch')
+    config.available_features.add('foundation')
+    config.available_features.add('objc_interop')
+else:
+    # TODO(yln): Works with the packaged swift distribution, but not during build.
+    #            We need to make libdispatch/foundation available in the test resource directory
+    #            or pass along the proper library include paths in the compiler invocations that are used
+    #            to build the tests.
+    # dir = make_path(config.swift_lib_dir, 'swift', config.target_sdk_name)
+    # prefix = config.target_shared_library_prefix
+    # extensions = [config.target_shared_library_suffix, '.a']
```

Comment:
Seems weird to hardcode ".a" here instead of having a target_static_library_suffix.